inst/include/oh_component.hpp

```diff
@@ -71,6 +71,8 @@ class OHComponent : public IModelComponent {
   double CNMVOC; // coefficent for NMVOC
   double CNOX;   // coefficent for NOX
   double CCH4;   // coefficent for CH4
+  unitval CH2;   // coefficent for CH4
```

Comment:
Do you mind correcting the spelling of "coefficient" here and in previous lines?

Reply:
Other questions: double vs unitval? Do we care? When should we use one over the other?

Reply:
> If it's used strictly within the component, and pretty isolated, then sometimes unitval is overkill -- feel free to use a double imho. If it's coming from user (INI or R) then having explicit units attached can be important. Does that help?

so all of the coefficients are read in via ini file/R.... I would assume that we would want to be consistent with out treatment of them but also don't think that any of these coefficients have units that are easy to interpret

Reply:
If it's used strictly within the component, and pretty isolated, then sometimes unitval is overkill -- feel free to use a double imho. If it's coming from user (INI or R) then having explicit units attached can be important. Does that help?